sdk/eventhubs/azure-messaging-eventhubs-checkpointstore-blob/src/main/java/com/azure/messaging/eventhubs/checkpointstore/blob/BlobCheckpointStore.java

```diff
@@ -219,14 +219,14 @@ public Flux<PartitionOwnership> claimOwnership(List<PartitionOwnership> requeste
                             LOGGER.atVerbose()
                                 .addKeyValue(PARTITION_ID_LOG_KEY, partitionId)
                                 .log(Messages.CLAIM_ERROR, error);
-                            return Mono.error(error);
+                            return Mono.empty();
```

Comment:
Here I only do the simple rollback. 
But if we want to permanently fix, we need to treat 412 status code error and some specific service response in different way, maybe just log the error in verbose level and return empty(). It return `Mono.error(error)` when it is an unexpected error.